doc/source/guides/common-tasks.rst

```diff
@@ -108,6 +161,14 @@ The code example below shows how to send the error report to Gini.
     GINIAPIManager *apiManager = gini.APIManager;
     BFTask *reportTask = [apiManager reportErrorForDocument:document.documentId summary:@"short summary" description:@"detailed description"];
 
+.. code-block:: swift
+    
+    // Assuming that `gini` is an instance of the `GiniSDK` facade class and
+    // `document` is an instance of the `GINIDocument` class as returned by `createDocumentWithFilename:fromImage:`.
+
+    let apiManager = gini.APIManager
+    let reportTask = apiManager?.reportErrorForDocument(document.documentId, summary: "short summary", description: "detailed description")
+
 Handling SDK errors
 ===================
 
```

Comment:
All errors that occur (not occure) while executing a task...You can react to those errors by....